pkg/security/securitycontextconstraints/matcher.go

```diff
@@ -20,38 +21,58 @@ import (
 	"github.com/openshift/origin/pkg/security/uid"
 )
 
-// SCCMatcher defines interface for SecurityContextConstraint matcher
```

Comment:
Why removing this comments instead?

Reply:
Because the only thing worse than no comments is incorrect comments and useless comments.